src/petals/utils/ping.py

```diff
@@ -16,7 +16,7 @@ async def ping(
     _dht: hivemind.DHT,
     node: hivemind.dht.DHTNode,
     *,
-    wait_timeout: float = 1,
+    wait_timeout: float = 5,
```

Comment:
Peers that use relay may yield large first ping, but it'll be smoothed in future.